tests/allure/src/main/java/org/mule/test/allure/AllureConstants.java

```diff
@@ -256,6 +256,15 @@
 
   }
 
+  interface VMFeature {
```

Comment:
- VM is not a proper maven test dependency
- The convention seems to be to have all these connector constants here.